packages/manager/modules/pci/src/projects/project/data-processing/submit-job/translations/Messages_fr_FR.json

```diff
@@ -30,5 +30,29 @@
   "data_processing_submit_job_error_message_odp_job_totalDriverMemory_tooBig": "Erreur de saisie : la somme de la mémoire et de l'overhead demandés pour le driver est trop importante",
   "data_processing_submit_job_error_message_odp_project_notEnoughQuotas": "Erreur de quotas : les quotas de votre projet Public Cloud sont insuffisants. Veuillez aller à la page suivante pour <a class=\"oui-link oui-link_icon\" href=\"{{ url }}\">gérer votre quota Public Cloud<span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a>",
   "data_processing_submit_job_error_message_odp_mainApplicationCode_notFound": "Erreur de saisie : le fichier demandé est introuvable sur le conteneur Object Storage que vous avez spécifié",
-  "data_processing_submit_job_error_message_odp_condaEnvironment_isRequired": "Erreur de saisie : aucun fichier d'environment Conda n'a été trouvé sur le conteneur Object Storage que vous avez spécifié"
+  "data_processing_submit_job_error_message_odp_condaEnvironment_isRequired": "Erreur de saisie : aucun fichier d'environment Conda n'a été trouvé sur le conteneur Object Storage que vous avez spécifié",
+  "data_processing_submit_job_spec_heading": "Votre job",
+  "data_processing_submit_job_spec_engine": "Type :",
+  "data_processing_submit_job_spec_engine_version": "Version {{version}}",
+  "data_processing_submit_job_spec_region": "Région :",
+  "data_processing_submit_job_spec_sizing": "Dimensionnement : ",
+  "data_processing_submit_job_spec_sizing_driver": "Driver :",
+  "data_processing_submit_job_spec_sizing_worker": "Executor :",
+  "data_processing_submit_job_spec_sizing_workers": "{{nbWorker}} × Executors :",
+  "data_processing_submit_job_spec_sizing_cores": "{{cores}} cœurs virtuels",
+  "data_processing_submit_job_spec_sizing_memory": "{{memory | bytes:0:false:'GB'}} mémoire",
```

Comment:
We have to avoid snippet code into content files.
you probably can use it like this :
`data-translate-values="{memory: (memoryValue | bytes:0:false:'GB')}"`

Reply:
Changed here: https://github.com/ovh/manager/pull/7047/commits/8876994d188ae6702e943be1d4cc1ac5482e6920